sympy/holonomic/tests/test_holonomic.py

```diff
@@ -719,7 +719,7 @@ def test_to_meijerg():
     p = hyper((-S(1)/2, -3), (), x)
     assert from_hyper(p).to_meijerg() == hyperexpand(p)
     p = hyper((S(1), S(3)), (S(2), ), x)
-    assert (hyperexpand(from_hyper(p).to_meijerg()) - hyperexpand(p)).expand() == 0
+    assert (hyperexpand(from_hyper(p).to_meijerg()) - hyperexpand(p)).simplify() == 0
```

Comment:
Full simplify shouldn't be used in tests. Instead find out which simplify function can work e.g. `cancel` in this case.

What is the underlying matrix multiplication that leads to a different result here?

The difference seems to be master:
```julia
In [2]: p = hyper((S(1), S(3)), (S(2), ), x)

In [3]: hyperexpand(p)
Out[3]: 
      x - 2      
─────────────────
(1 - x)⋅(2⋅x - 2)
```
vs the PR
```julia
In [11]: hyperexpand(p)
Out[11]: 
     x  
 1 - ─  
     2  
────────
       2
(x - 1)
```